### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by herbeck558**

### CODE REVIEW

**Review by dlangst — COMMENTED** [review]
Few comments:
1. Typo in 'cartidges' here:  Deploying and updating the ACD provided cartidges
2. Should be 'update' and 'ACD' (optional: could clarify as 'these predefined cartridges' and 'this supertenant tenant ID' in this sentence:  An administrator can deploy or updated these by specifying this tenant on a direct call to an acd container as documented below.
3. Typo 'too' here: In a new terminal window run one the following commands to deploy the cartridge. Be sure to be in the directory where the cartridge zip file was downloaded too.
4. Remove 'one' here:  In a new terminal window run one the following commands to deploy the cartridge.
5. I think the following sentence should be 'NOTE: Use POST to create it the first time, or PUT to update a previous cartridge deployment':  NOTE: Use POST to create it the first time, or PUT to update it later.
6. Use 'APIs' here: Test ACD apis to verify the cartridge was deployed using an externalized route to the ACD service.
7. Typo 'fornt' here:  In the examples below replace <route_host> with the hostname and path to the externalized route to ACD or if you have a security proxy configured in fornt of ACD...

File: src/pages/usage/getting-started.md [hunk 1]
```diff
@@ -30,8 +30,12 @@ Port-forward to an ACD pod. **Note that only an admin can do the port-forward an
 oc port-forward ${pod_name} -n ${acd_namespace} 9443:9443
 ```
 
+The terminal should hang here waiting to accept network traffic on the localhost port 9443.  Open a new terminal window to run the following steps.
+
 #### 3. Use curl to deploy the cartridge.
 
+In a new terminal window run one the following commands to deploy the cartridge.  Be sure to be in the directory where the cartridge zip file was downloaded too.
```
Comment (cagrev) [inline src/pages/usage/getting-started.md:37]: Drop "too" at the end, or add "to which" and drop "where" in phrase "where the cartridge zip file was downloaded."

File: src/pages/usage/getting-started.md [hunk 2]
```diff
@@ -41,9 +45,9 @@ curl -k -X POST \
...
 
-The example shows the default `X-Watson-UserInfo` header to specify the supertenant tenant id required to deploy or change these ACD provided cartridges to the shared tenant configuration location. This is the default header and format used by ACD if no Tenant header (tenantHeader) value was set in the ACD deployment to support tenant isolation use that header and remove the bluemix-instance-id prefix.  If you set a Tenant header value on the ACD instance you will use that header instead.  For example, if using the OAuth proxy header with tenancy:
+The command above uses the default `X-Watson-UserInfo` header to specify the supertenant tenant id required to deploy or change these provided cartridges to the shared tenant configuration location. This is the default header and format used by ACD if no Tenant header (tenantHeader) value was set in the ACD deployment to support tenant isolation.   If you set a Tenant header value on the ACD instance you will use that header instead.  Using the OAuth proxy and the mulitenancy with the X-Forwarded-User as the tenantHeader you'd use this command instead with a different header.
```
Comment (cagrev) [inline src/pages/usage/getting-started.md:50]: add "path" for the multitenancy path with the ... , or drop the word "the" in front of multitenancy

File: src/pages/usage/getting-started.md [hunk 3]
```diff
@@ -54,10 +58,10 @@ curl -k -X POST \
...
 
 **Deploying custom cartridges**
-If you are deploying a custom cartridge rather than an ACD provided catridge you would remove this header completely to have it placed into the defaultTenant configuration location or if you are using ACD multitenancy (which requires a security proxy) you should deploy and update your custom cartridges through the proxy route using the bearer token for the tenant and not use a port forward direct to an ACD container at all for deploying custom catridges.
+If you are deploying a custom cartridge rather than an ACD provided catridge you would remove this header completely to have it placed into the defaultTenant configuration location or if you are using ACD multitenancy (which requires a [security proxy](../../security/manage-access)) you should deploy and update your custom cartridges through the proxy route using the bearer token for the tenant and not use a port forward direct to an ACD container at all for deploying custom catridges.
```
Comment (cagrev) [inline src/pages/usage/getting-started.md:64]: Do we define anywhere what the defaultTenant configuration location is? That should probably be spelled out to make it more.

**Review by cagrev — APPROVED** [review]

**Pull request merged**